linode/vpc/framework_resource.go

```diff
@@ -56,6 +57,10 @@ func (r *Resource) Create(
 	}
 
 	data.FlattenVPC(ctx, vpc, true)
+
+	// IDs should always be overridden during creation (see #1085)
+	data.ID = types.StringValue(strconv.Itoa(vpc.ID))
```

Comment:
@zliang-akamai Would it make sense to instead remove the preserveKnown logic for the ID field in the `data.FlattenX(...)` methods? I'm hesitant because it might introduce some unintuitive behavior, but might simplify this workaround a bit.

Reply:
Yep, both ways seem to be fine to me